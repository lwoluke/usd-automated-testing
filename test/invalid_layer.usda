#usda 1.0
(
    defaultPrim = "Root"
    subLayers = [
        @missing_layer.usda@  # Invalid: this sublayer does not exist
    ]
)

def "Root"
{
    def Xform "Geom"
    {
        def Mesh "Cube"
        {
            int[] faceVertexCounts = [4, 4, 4, 4, 4, 4]
            int[] faceVertexIndices = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]
            point3f[] points = [
                (-1, -1, -1), (1, -1, -1), (1, 1, -1), (-1, 1, -1),
                (-1, -1, 1), (1, -1, 1), (1, 1, 1), (-1, 1, 1)
            ]
        }
    }

    def "Shaders"
    {
        def Shader "SimpleShader"
        {
            token info:id = "UsdPreviewSurface"
            color3f inputs:diffuseColor = (0.5, 0.5, 0.5)
        }
    }
}
