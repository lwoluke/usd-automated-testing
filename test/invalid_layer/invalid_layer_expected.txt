Opened USD file Successfully.

[FAIL] Validate Geometry: Geometry validation failed with the following issues:
- Invalid extent bounds at: /Root/Geom/Cube

[PASS] Validate Shaders: All shaders and their connections are valid.
[FAIL] Validate Layer Structure: Layer structure validation failed with the following issues:
- Unresolved sublayer: missing_layer.usda

[PASS] Validate Variants: No variants found in the scene. That's acceptable.

Summary:
  Passed: 2
  Failed: 2

Some tests failed. Please review the USD file and address the failing tests.
